### PR TITLE
Py working sim

### LINKED ISSUE #117: Bug in spatial simulation flow
**Describe the bug**
Spatial simulation flow doesn't run - error in distance metric (passing cell not location)

Further issues when:

- [ ] Trying to set infected individuals in a single cell (can't)
- [ ] Reading in population with infected individuals (doesn't update time to status change)

### PR DESCRIPTION
**Pull request by KCGallagher**
Fixes bugs in the simulation sweeps, and adds useful little things.

* Resolves error in spatial simulation flow where distance metric is passed cell not location
* Enables infected individuals to be put into in a single cell in the initial infected sweep (not spread across pop)
* Fixes reading in population with infected individuals (updates next status, time to status change and infectiousness)

Closes #117

### CODE REVIEW

**Review by rcw5890 — CHANGES_REQUESTED** [review]

File: pyEpiabm/pyEpiabm/routine/file_population_config.py [hunk 1]
```diff
@@ -93,8 +96,14 @@ def make_pop(input_file: str, random_seed: int = None):
...
-                    new_microcell.add_people(int(line[column]),
-                                             InfectionStatus(value))
+                    for i in range(int(line[column])):
+                        person = Person(new_microcell)
+                        person.infection_status = InfectionStatus(value)
+                        new_microcell.add_person(person)
+                        host_sweep.update_next_infection_status(person)
+                        host_sweep.update_time_status_change(person, 0)
+                        if str(person.infection_status).startswith('Infect'):
+                            HostProgressionSweep.set_infectiousness(person, 0)
```
Comment (I-Bouros) [inline pyEpiabm/pyEpiabm/routine/file_population_config.py:106]: I am a bit confused: If the person's infectious status is a type of infectious, why would you set the infectiousness to 0 in the host progression sweep?

**Review by I-Bouros — CHANGES_REQUESTED** [review]
Couple of comments. And a suggestion

File: pyEpiabm/pyEpiabm/routine/file_population_config.py (hunk 1 above)
Comment (KCGallagher) [reply pyEpiabm/pyEpiabm/routine/file_population_config.py]: This had me confused for ages as well! Turns out that zero isn't the infectiousness, its the time you are setting the value at - the actual value is sampled from infectiousness. I've tidied it up to make that more obvious though

File: README.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ Configure a simulation with a number of parameters. These are split into two cat
 * `simulation_start_time`: The initial time at the start of the simulation
 * `simulation_end_time`: The final time at which to stop the simulation
 * `initial_infected_number`: The initial number of infected individuals in the population
+* `initial_infect_cell`: Whether to choose initial infectied indivudals from a single cell
```
Comment (rcw5890) [inline README.md:84]:
```suggestion
* `initial_infect_cell`: Whether to choose initial infected individuals from a single cell
```

File: pyEpiabm/pyEpiabm/core/microcell.py [hunk 3]
```diff
@@ -61,6 +61,22 @@ def set_id(self, id):
         """
         self.id = id
 
+    def add_person(self, person):
+        """Adds :class:`Person` with given :class:`InfectionStatus`
+        to Microcell.
+
+        Parameters
+        ----------
+        person: Person
```
Comment (rcw5890) [inline pyEpiabm/pyEpiabm/core/microcell.py:70]:
```suggestion
        person : Person
```

**Review by rcw5890 — CHANGES_REQUESTED** [review]

**Review by I-Bouros — APPROVED** [review]

**Review by rcw5890 — APPROVED** [review]

**Pull request merged**